ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -30,23 +31,30 @@
 public class NetworkMetrics {
```

Comment:
In general, would prefer grouping these variables by component, rather than by type.